translator/server.py

```diff
@@ -12,11 +12,14 @@
 from transformers import MarianMTModel, MarianTokenizer
 from typing import Sequence
 from libargos import install_packages
+import socket
 
 PORTS = [8000, 5000]

```

Comment:
Maybe add more ports. If we have 8-10 or so ports, theres no scenario except for testing where all ports will be used up.